### ISSUE TITLE
Untranslatable text in the dialog

### ISSUE THREAD

**Issue by adem4ik** [FelixdelasPozas/TrayWeather#52]
Version 1.10.0
Windows 10 x64

It is hard to reproduce. I guess you can see this dialog, if you press on the tray icon during long weather request.

Screenshot: 
![image](https://user-images.githubusercontent.com/4707112/127439840-f7799aeb-c460-469c-8e2b-3e569f25f5a4.png)
Russian translation:
`TrayWeather запросил погодные данные для вашего географического местоположения и всё ещё ожидает ответа.`

**Comment by FelixdelasPozas** [comment]
Hi @adem4ik .
I've updated the language file with the error message and activated the translation earlier in the execution to also translate other errors. The untranslated error messages are:

```
* TrayWeather cannot execute in this computer because there isn't a tray available!. The application will exit now.
* TrayWeather is already running!
* TrayWeather cannot execute without a valid location and a valid OpenWeatherMap API Key. The application will exit now. 
```

Some errors will never be shown in the translated language (like the one if the system tray doesn't exist) as the user will never be able to execute the configuration dialog in that case, but let's cover all bases. Thanks in advance.

Regards,
Félix.

**Comment by adem4ik** [comment]
Russina translation:
```
* TrayWeather не может запуститься на этом компьютере, потому что трей не доступен! Приложение сейчас завершит работу.
* TrayWeather уже запущен!
* TrayWeather не может запуститься без действительных местоположения и ключа API OpenWeatherMap. Приложение сейчас завершит работу.
```

**Comment by FelixdelasPozas** [comment]
Thanks! For the next release I'll need a new translation: 

```
TrayWeather needs to be restarted for the new configuration to take effect. The application will exit now.
```

Once that is translated I'll release the version 1.10.1.

Regards,
Félix.

**Comment by adem4ik** [comment]
`TrayWeather needs to be restarted for the new configuration to take effect. The application will exit now.`
->
`TrayWeather надо перезапустить, чтобы применить новые настройки. Приложение сейчас завершит работу.`

**Comment by FelixdelasPozas** [comment]
Many thanks. Released [1.10.1](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.10.1) with the updated translations.

Regards,
Félix.
